jcl/src/java.base/share/classes/jdk/internal/misc/Unsafe.java

```diff
@@ -5864,6 +5864,29 @@ public void invokeCleaner(ByteBuffer bbo) {
 	}
 /*[ENDIF] Java12 */
 
+/*[IF Java14]*/
+	/**
+	 * Make sure that the virtual memory at address "addr" for length "len" has been flushed to physical memory.
+	 * Throw RuntimeException if cache flushing not enabled on the runtime OS.
+	 * 
+	 * Temporary stub method
+	 */
+	public void writebackMemory(long addr, long len) {
+		if (!isWritebackEnabled()) {
+			throw new RuntimeException("Cache flushing not enabled on this platform."); //$NON-NLS-1$
+		}
```

Comment:
This should throw an UnsupportOperationException or a RuntimeException to show that the methods aren't implemented yet.  By ignoring the call, it makes it hard to know whether this is implemented or not.  Better to explicitly throw until we have an implementation